Sources/docc/DocCDocumentation.docc/Reference Syntax/API Reference Syntax/Metadata.md

```diff
@@ -42,7 +42,7 @@ Use the `Metadata` directive with the ``TitleHeading`` directive to configure th
 }
 ```
 
-Starting with version 5.11, use the `Metadata` directive with one or more ``Redirected`` directives
```

Comment:
Should we also update this comment in the source code?
https://github.com/apple/swift-docc/blob/d3043178e3ca18b6113e0420568cc5fc3cbd34d6/Sources/SwiftDocC/Semantics/Redirect.swift#L24-L25

(I think this will necessitate regenerating the symbol graph again)

Reply:
Yes, thank you. I don't know why I missed that one.

I fixed this in https://github.com/apple/swift-docc/pull/936/commits/54c4ae5ce504e1e2d703262550efd834b8409414